src/rpp/stack_trace.cpp

```diff
@@ -972,10 +977,15 @@ namespace rpp
 
         explicit operator bool() const noexcept { return error == nullptr; }
 
-        ThreadContext() noexcept
+        ThreadContext(HANDLE thread = nullptr) noexcept
```

Comment:
See comment below, would make sense to use `uint64_t threadId = 0` instead and open thread based on thread ID:
```cpp
if (threadId == 0)
{
    hThread = getCurrentThread();
}
else
{
    DWORD flags = THREAD_GET_CONTEXT | THREAD_QUERY_INFORMATION | THREAD_SUSPEND_RESUME;
    hThread = OpenThread(flags, FALSE, threadId);
    ownsHandle = true; // you need to CloseHandle() in ThreadContext destructor after using OpenThread
}
```